tools/export/sw4stm32/__init__.py

```diff
@@ -14,7 +14,9 @@
 See the License for the specific language governing permissions and
 limitations under the License.
 """
-from os.path import splitext, basename, join
+import os
+
+from os.path import splitext, basename, join, dirname
```

Comment:
Is there a specific reason to `import os; from os.path import XX` instead of doing directly `from os.path import XX` as it was before ?